CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# How to contribute
+
+Thank you for your interest in contributing to the AeroGear project. We want
+keep this process as easy as possible so we've outlined a few guidelines below. 
+For more information about the AeroGear project, visit 
+[our website](https://aerogear.org/).
+
+## Asking for help
+
+Whether you're contributing a new feature or bug fix, or simply submitting a
+ticket, the AeroGear team is available for technical advice or feedback. 
+You can reach us at #aerogear on [Freenode IRC](https://freenode.net/) or the 
+[aerogear-dev list](https://www.redhat.com/mailman/listinfo/aerogear-dev) 
```

Comment:
wrong mailing list archive address.

It's here: https://lists.jboss.org/mailman/listinfo/aerogear-dev